purchase_supplier_postage_free/model/purchase_order.py

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
+#
+#
+#    Authors: Guewen Baconnier
+#    Copyright 2015 Camptocamp SA
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+#
+
+import openerp.addons.decimal_precision as dp
+from openerp import models, fields, api
+
+
+class PurchaseOrder(models.Model):
+    _inherit = 'purchase.order'
+
+    free_postage = fields.Float(
```

Comment:
Hi.

I think that the free portage should not be computed, but filled with an @api.onchange on the partner_id. For the time being, if a user change the free_postage of a partner, it will change the free_postage of all the old purchase order, and the value free_postage_reached (stored) will change, that is wrong.
In many cases, free postage is a value that can change during the time.

Don't you think ?

(or maybe I missed something)